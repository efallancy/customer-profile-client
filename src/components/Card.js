import styled from 'styled-components';

const Card = styled.div`
  background: #FFFFFF;
  box-shadow: 0 2px 48px 0 rgba(176,192,237,0.50);
  border-radius: 12px;
  padding: 2rem;
  margin: 1rem 0;
`;

export default Card;
