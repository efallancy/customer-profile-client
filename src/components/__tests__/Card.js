import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Card from '../Card';

describe('Card', () => {
  it('renders correctly', () => {
    const card = renderer.create(<Card />).toJSON();
    expect(card).toMatchSnapshot();
  });
});
