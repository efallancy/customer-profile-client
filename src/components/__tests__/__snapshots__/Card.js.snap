// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
.c0 {
  background: #FFFFFF;
  box-shadow: 0 2px 48px 0 rgba(176,192,237,0.50);
  border-radius: 12px;
  padding: 2rem;
  margin: 1rem 0;
}

<div
  className="c0"
/>
`;
